print.c

```diff
@@ -475,7 +475,9 @@ lkup_port(p, pr, src)
 	MALLOC_S nl;
 	char *nm, *pn;
 	static char pb[128];
+#if	!defined(HASNORPC_H)
 	static int pm = 0;
```

Comment:
Instead of having this here and then checking it later to see if we should call fill_portmap(), how about we move that static variable to fill_portmap() and check inside of the function whether or not it should run? I can send a separate patch for that, if you want to remove this part of this commit.

Reply:
I cherry-picked your change.